lib/chef/provider/package/apt.rb

```diff
@@ -138,20 +149,12 @@ def upgrade_package(name, version)
         end
 
         def remove_package(name, version)
-          if name.is_a?(Array)
-            package_name = name.join(' ')
-          else
-            package_name = name
-          end
+          package_name = [ name ].flatten.join(' ')
```

Comment:
If it was still written the way that I replaced, I'd agree.  As it is, its clear and pretty concise and makes the implementation really obvious.

I want to do a style rage-fix cleanup after 12.1.0 ships, anyway, though, and want to step back and see if there might not be a better way to do this (e.g. maybe the subclass should expose remove_package_array() instead of remove_package() and the superclass will be responsible for doing `respond_to?(:remove_package_array)` and coercing the argument into an array, then this line goes away -- all this array-or-scaler argument stuff is messy and I'd like it more contained).